### DISCUSSION TITLE
Updating quill_controller.js

### DISCUSSION THREAD

**Discussion by maatata** [orchidsoftware/platform#1734] [Q&A]
Hi

I have been able to publish Orchid assets by the following command:
```php artisan vendor:publish```

All good and all the views and JS and CSS files are added to Laravel and if I need to amend anything I can simply do it on the published files rather than touching Orchid vendor files.

However I'm trying to amend Javascript files which are in the following path
`resources\js\orchid\controllers` 
specifically the file called `quill_controller.js`

I have amended that file and saved it but nothing has changed on the page as I refresh the page. I tried clearing Laravel cache but still the same. Currently Orchid is compiling a file called `orchid.js` and it's putting all the controller files together and put it in this `orchid.js` file but it does not update if I change any of those controller files.

Could you please let me know what step I need to take or what command I need to run to recompile that `orchid.js` file and grab the changes I made on the published files?

That would be very helpful if you could help.

Thanks.

**Comment by danielcatao** [comment]
I'm facing the same problem but with relation_controller.js I'm trying to console.log the const chunk when a request is made

**Reply by danielcatao** [reply]
I've found a solution for my case and posted on discord help channel.

**Reply by macit-emre** [reply]
can you please post here too?

**Reply by danielcatao** [reply]
In order to debug relation_controller.js you need to use npx mix inside vendor/orchid folder it will regenerate vendor.js and orchid.js

**Reply by macit-emre** [reply]
that helped me a lot. Thanks. a small addition,
```
- cd vendor/orchid/platform
- then npx mix -> (if gives error, then these) 
- - npm update 
- - if missing  node-sass module -> npm install --save-dev  --unsafe-perm node-sass
- - npx update
- then php artisan vendor:publish --tag=laravel-assets --ansi --force
```
